src/app/about-us/about-us.component.html

```diff
@@ -1,7 +1,8 @@
 <gp-page-banner [title]="'about-us.who.title' | translate" [description]="'about-us.who.description' | translate" 
 picture="assets/images/about-us.JPG"> 
 </gp-page-banner>
-<div class="container">
+<div class="container">  
+  <gp-mission-vision></gp-mission-vision>     
```

Comment:
To keep consistency, you should put mission-vision inside a div with class "section-space" here, instead of using the class inside the component, like gp-members component is doing it.